### DISCUSSION TITLE
Turn off the distracting blue notifications dot

### DISCUSSION THREAD

**Discussion by Sjors** [community/community#40346] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I've asked support about this in 2019, but so far it hasn't been implemented.

The blue notifications dot on the top right is a massive distraction. Whenever I'm focussed on reviewing a pull request, it suddenly appears, like an SMS on my phone. It won't go away until I've marked all notification(s) as seen. This forces me to context switch needlessly.

I'd like the option to disable the blue dot. The `/notifications` page should still work, but I want to visit it at a time of my choosing.

I'm sure I could spend a year in a silent meditation retreat and train myself to not notice the blue dot. But that seems more involved than simply having an option to get rid of it :-)

**Comment by ossanna16** [comment]
Hi there @Sjors and welcome to our community! Thank you for the feedback!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Sjors! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by justinmchase** [comment]
I would like an option to turn the blue dot red, so its extra distracting.

**Comment by only1thor** [comment]
As a workaround, try using an adblocker to block the element. 
I do this with uBlock origin, and use pre-defined filters as bookmarks to open the context i want to avoid getting distracted by notifications outside my current context.
